src/oss/schemas/RemoteConnections.yml

```diff
@@ -0,0 +1,7 @@
+type: object
+properties:
+  # TODO(danmoran): Add pagination / links?
```

Comment:
None of our other APIs support pagination that I know of so I think it's fine to leave this as a TODO for now. Pagination support here seems like a lower priority than it would be in some other areas as well.